client/scss/components/_tabs.scss

```diff
@@ -116,53 +9,72 @@
   }
 }
 
-@include media-breakpoint-up(sm) {
-  .tab-nav {
-    // For cases where tab-nav should merge with header
-    &.merged {
-      @apply w-bg-grey-50;
-    }
-
-    li {
-      width: auto;
-      padding: 0;
-    }
-
-    a {
-      padding-inline-start: $mobile-nice-padding;
-      padding-inline-end: $mobile-nice-padding;
-    }
-
-    li.settings a {
-      padding-inline-start: 2em;
-      padding-inline-end: 2em;
+.w-tabs {
+  &__wrapper {
+    @apply w-mb-10 w-overflow-x-auto w-scrollbar-thin;
+  }
+
+  &__list {
+    @include nice-padding();
+    @apply w-flex w-my-[3px] w-space-x-6 w-border-b w-border-grey-100 w-w-fit;
+  }
+
+  &__tab {
+    @apply w-label-3
+    w-box-border
+    w-inline-flex
+    w-text-grey-400
+    hover:w-text-primary
+    w-whitespace-nowrap
+    w-py-4
+    w-font-medium
+    w-relative
+    after:w-block
+    after:w-w-0
+    after:w-h-[2px]
+    after:w-bg-primary
+    after:w-absolute
+    after:w-left-0
+    after:-w-bottom-px
+    after:w-transition-all
+    motion-reduce:after:w-transition-none
+    hover:after:w-w-full;
+
+    &[aria-selected='true'] {
+      @apply after:w-w-full w-text-primary;
     }
   }
 
-  .modal-content .tab-nav li {
-    padding: 0;
-    min-width: 0;
-
-    &:first-of-type {
-      padding-inline-start: $desktop-nice-padding;
+  &__errors {
+    @apply w-hidden
+    w-box-border
+    w-w-4
+    w-h-4
+    w-text-[0.75rem]
+    w-flex
+    w-justify-center
+    w-items-center
+    w-font-bold
+    w-bg-critical-200
+    w-text-white
+    w-border
+    w-border-white
+    w-rounded-full
+    w-absolute
+    w-top-[7px]
+    -w-right-[15px];
```

Comment:
Should these be rem also ?

Need to be careful we don't break zoom in / out

Reply:
I've made this change now